### ISSUE TITLE
PR#9 breaks something

### ISSUE THREAD

**Issue by fujiisoup** [flexible-atomic-code/fac#10]
I think PR#9 breaks some parts of the code.
@mfgu , can you investigate this?

To reproduce this, do
```
cd demo/structure
make
```
It works until PR#9 was merged.

**Comment by mfgu** [comment]
it ran fine on mine. what error did you see?

**Comment by fujiisoup** [comment]
`make[1]: *** [all] Segmentation fault (core dumped)`

It also fails on the automatic testing system, which makes a clean install of fac and testings it every time.
It builds fac by the following sequence of the commands
```
./configure --prefix=$pwd$
make clean
make
make install
make pfac
make install-pfac
```

The full log is here
https://travis-ci.org/flexible-atomic-code/fac/builds/361910488?utm_source=github_status&utm_medium=notification

Maybe it works if openmp enabled?

**Comment by mfgu** [comment]
i see. will fix it in the next PR.

**Comment by fujiisoup** [comment]
Thanks a lot :)

The test is already running, so you can see the testing status by an icon next to the commit id.

**Comment by fujiisoup** [comment]
Closed via #12
